hub/policy/src/test/java/uk/gov/ida/hub/policy/domain/controller/CountrySelectedStateControllerTest.java

```diff
@@ -170,7 +170,7 @@ public void shouldValidateIncorrectLoa() {
         exception.expect(StateProcessingValidationException.class);
         exception.expectMessage(
                 String.format("Level of assurance in the response does not match level of assurance in the request. Was [%s] but expected [%s].",
-                        LevelOfAssurance.LEVEL_1, ImmutableList.of(LevelOfAssurance.LEVEL_2)));
+                        LevelOfAssurance.LEVEL_1, ImmutableList.of(LEVEL_2)));
```

Comment:
Being very picky, but we should probably either statically import both LevelOfAssurance.LEVEL_1 and LevelOfAssurance.LEVEL_2 or neither.

Reply:
Fixed 👍